scheduler/server/cluster_state.go

```diff
@@ -237,25 +237,25 @@ func (c *clusterState) update(updates []cluster.NodeUpdate) {
 			adds += 1
 			// UserInitiated is true only when this NodeUpdate comes from a ReinstateWorker or OfflineWorker request
 			if update.UserInitiated {
-				log.Infof("Reinstating node %s", update.Id)
+				log.Infof("NodeAdded: Reinstating node %s", update.Id)
```

Comment:
adding common Add, Remove, Suspend qualifiers to log statements to make it easier to trace a node's handling in the log